backend/src/main/java/com/redhat/cloud/notifications/config/BackendConfig.java

```diff
@@ -152,17 +153,19 @@ public boolean isInstantEmailsEnabled() {
      * @return {@code true} if we are allowed to delete integrations from the
      * Kessel inventory.
      */
-    public boolean areKesselInventoryIntegrationRemovalsEnabled() {
+    public boolean areKesselInventoryIntegrationRemovalsEnabled(String orgId) {
         if (unleashEnabled) {
-            return unleash.isEnabled(kesselInventoryIntegrationsRemovalToggle, false);
+            UnleashContext unleashContext = buildUnleashContextWithOrgId(orgId);
+            return unleash.isEnabled(kesselInventoryIntegrationsRemovalToggle, unleashContext, false);
         } else {
             return kesselInventoryIntegrationRemovalsEnabled;
         }
     }
 
-    public boolean isKesselInventoryEnabled() {
+    public boolean isKesselInventoryEnabled(String orgId) {
         if (unleashEnabled) {
-            return unleash.isEnabled(kesselInventoryToggle, false);
+            UnleashContext unleashContext = buildUnleashContextWithOrgId(orgId);
+            return unleash.isEnabled(kesselInventoryToggle, unleashContext, false);
         } else {
             return kesselInventoryEnabled;
         }
```

Comment:
Also I am not sure if we can disable the inventory per organization ID :disappointed: , because if we migrate all of our integrations to Kessel, and we enable the inventory integration for the org id `acme`, we would only be creating the integrations for them...